src/tracing/errors.test.ts

```diff
@@ -29,4 +29,141 @@ describe('filterKnownErrors', () => {
     const originalException = new SyntaxError("Unexpected token '<'")
     expect(filterKnownErrors(ERROR, { originalException })).toBe(null)
   })
+
+  describe('chunk errors', () => {
+    afterEach(() => {
+      jest.restoreAllMocks()
+    })
+
+    it('filters 499 error coded chunk error', () => {
+      const mockGetEntriesByType = jest.fn(() => {
+        return [
+          {
+            name: 'https://app.uniswap.org/static/js/20.d55382e0.chunk.js',
+            entryType: 'resource',
+            startTime: 253.79999999701977,
+            duration: 3.0999999940395355,
+            initiatorType: 'link',
+            nextHopProtocol: 'h2',
+            renderBlockingStatus: 'non-blocking',
+            workerStart: 254,
+            redirectStart: 0,
+            redirectEnd: 0,
+            fetchStart: 254.19999998807907,
+            domainLookupStart: 254.19999998807907,
+            domainLookupEnd: 254.19999998807907,
+            connectStart: 254.19999998807907,
+            secureConnectionStart: 254.19999998807907,
+            connectEnd: 254.19999998807907,
+            requestStart: 255.09999999403954,
+            responseStart: 256,
+            responseEnd: 256.8999999910593,
+            transferSize: 0,
+            encodedBodySize: 227688,
+            decodedBodySize: 227688,
+            responseStatus: 499,
+            serverTiming: [],
+          },
+        ] as any
+      })
+      window.performance.getEntriesByType = mockGetEntriesByType
+      const originalException = new Error(
+        'Loading chunk 20 failed. (error: https://app.uniswap.org/static/js/20.d55382e0.chunk.js)'
+      )
+      expect(filterKnownErrors(ERROR, { originalException })).toBeNull()
+    })
+
+    it('keeps error when status is different than 499', () => {
+      const mockGetEntriesByType = jest.fn(() => {
```

Comment:
Instead of defining a mock and then assigning it, do it in one line. It's easier to read / keeps it all in one place:

```
jest.mock(window.performance, 'getEntriesByType').mockResolves([...])
```

Reply:
I tried doing this but getEntriesByType isn't defined in this execution env so that won't work properly

Reply:
i did it on one line now instead tho